app/code/core/Mage/Adminhtml/Block/Cache.php

```diff
@@ -37,23 +39,24 @@ public function __construct()
         $this->_removeButton('add');
         $this->_addButton('flush_magento', [
             'label'     => Mage::helper('core')->__('Flush OpenMage Cache'),
-            'onclick'   => 'setLocation(\'' . $this->getFlushSystemUrl() . '\')',
+            'onclick'   => JsHelper::getSetLocationJs($this->getFlushSystemUrl()),
```

Comment:
Not sure about this change to use helper class directly, doing this will make it not overridable..

Reply:
Going to revert this.